addOns/exim/src/test/resources/org/zaproxy/addon/exim/log/fourEqualsNoSpace.txt

```diff
@@ -0,0 +1,59 @@
+====1 ==========
```

Comment:
IMO these should be trimmed to have only the minimum data required (we are testing the delimiters not the contents of the messages).

Reply:
`User-Agent`, `Accept`, and `Accept-Language` can also be removed.